sale_invoice_plan/views/sale_view.xml

```diff
@@ -160,7 +160,10 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">sale.invoice.plan</field>
         <field name="view_mode">tree</field>
-        <field name="view_id" ref="view_sale_invoice_plan_tree_readonly" />
+        <field
+            name="view_id"
+            ref="sale_invoice_plan.view_sale_invoice_plan_tree_readonly"
```

Comment:
There's no need here of this. Only the first one. In fact, here it's advisable to use relative XML-IDs.